### PR TITLE
adds more server logic

### PR DESCRIPTION
**Pull request by mfbEngagency**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -13,5 +12,6 @@
     "express": "^4.17.2",
     "nodemon": "^2.0.15",
     "pg": "^8.7.1"
-  }
+  },
+  "description": ""
```
Comment (wm-engagency) [inline server/package.json:16]: This seems superfluous

**Review by wm-engagency — APPROVED** [review]
🚀

File: server/package.json (hunk 1 above)
Comment (mfbEngagency) [reply server/package.json:16]: you didn't give me a chance to fix it

Comment (wm-engagency) [reply server/package.json:16]: 🙄

**Pull request merged**